examples/i2c/src/main.xc

```diff
@@ -60,18 +69,15 @@ int main(void)
 {
   i2c_slave_callback_if i_i2c;
   interface control i_control[1];
```

Comment:
why i_control is an array [1], could we let this as plain interface?

Reply:
The API requires an array of control interface, see an example [here](https://github.com/xmos/lib_device_control/blob/develop/lib_device_control/api/control.h#L75). I would leave the code as it is to match the current API.